### ISSUE TITLE
Support Edge Dictionaries

### ISSUE THREAD

**Issue by alexmuller** [fastly/fastly-ruby#66]
It would be really useful for us if this gem supported the [Edge Dictionary](https://docs.fastly.com/guides/edge-dictionaries/) API methods. Do you have any plans to implement them in this gem? If we were able to would you review a pull request to add them?

**Comment by alexmuller** [comment]
I think I'm looking for something a bit like this but I'm still getting into the code in this repo for the first time:

```diff
diff --git a/lib/fastly.rb b/lib/fastly.rb
index 9db95a2..7fd0da7 100644
--- a/lib/fastly.rb
+++ b/lib/fastly.rb
@@ -13,6 +13,7 @@ require 'fastly/backend'
 require 'fastly/cache_setting'
 require 'fastly/condition'
 require 'fastly/customer'
+require 'fastly/dictionary'
 require 'fastly/director'
 require 'fastly/domain'
 require 'fastly/header'
@@ -137,7 +138,7 @@ class Fastly
     client.get_stats('/stats/regions')
   end
 
-  [User, Customer, Backend, CacheSetting, Condition, Director, Domain, Header, Healthcheck, Gzip, Match, Origin, RequestSetting, ResponseObject, Service, S3Logging, Syslog, VCL, Version].each do |klass|
+  [User, Customer, Backend, CacheSetting, Condition, Dictionary, Director, Domain, Header, Healthcheck, Gzip, Match, Origin, RequestSetting, ResponseObject, Service, S3Logging, Syslog, VCL, Version].each do |klass|
     type = Util.class_to_path(klass)
 
     # unless the class doesn't have a list path or it already exists
diff --git a/lib/fastly/dictionary.rb b/lib/fastly/dictionary.rb
new file mode 100644
index 0000000..532ae93
--- /dev/null
+++ b/lib/fastly/dictionary.rb
@@ -0,0 +1,5 @@
+class Fastly
+  class Dictionary < BelongsToServiceAndVersion
+    attr_accessor :service_id
+  end
+end
diff --git a/lib/fastly/version.rb b/lib/fastly/version.rb
index 3d34e06..adea00c 100644
--- a/lib/fastly/version.rb
+++ b/lib/fastly/version.rb
@@ -144,6 +144,16 @@ class Fastly
       hash.nil? ? nil : hash
     end
 
+    def dictionaries
+      raise 'Not implemented'
+      # fetcher.client.get("#{Dictionary.get_path(service_id, number, '')}")
+      fetcher.client.get("#{Dictionary.list_path}")
+    end
+
+    def dictionary(name)
+      raise 'Not implemented'
+    end
+
     # Validate this Version
     def validate
       hash = fetcher.client.get("#{Version.put_path(self)}/validate")
```

**Comment by kimmi8** [comment]
Eventually yes. Thanks for notifying us about this.

**Comment by icco** [comment]
This is done btw.

**Comment by lanej** [comment]
Fixed by #69 #86
